test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -819,6 +820,74 @@ ava('._prependLegCollectionWithRouteAmendment() adds LegModels for each routeStr
     t.true(fms.legCollection[1].routeString === routeAmmendmentMock[1]);
 });
 
+ava('._regenerateSidLeg() creates a new SID leg of identical route string for the currently assigned departure runway', (t) => {
+    const initialRunwayMock = airportModelFixture.getRunway('19L');
```

Comment:
I believe this should be called a _fixture_ because this is an actual `RunwayModel` 

`mock`: [fake] data that makes up a specific object or item in an app
`fixture`: an instantiated class, with working methods, that can be used under test